.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules
 dist
+.idea
```

Comment:
please include this in your *user* gitignore, this has nothing to do in a *project* gitignore